docs/source/experimental/warning.rst

```diff
@@ -12,3 +12,4 @@ The tools described in this section are actively being designed and are subject
 
    Navigator Console </tools/navigator/console>
    Extractor </tools/extractor>
+   Visualization </tools/visual>
```

Comment:
I don't believe in experimental tools - they are just tools you are saying are a bit crappy. Can we just put it as a normal tool.

Reply:
Experimental is only for stuff that we're not really happy to support properly. Everything else should just be in the normal tools section

Reply:
Cool will remove it from this warning page